lib/ansible/plugins/callback/default.py

```diff
@@ -106,7 +113,7 @@ def v2_runner_on_ok(self, result):
             self._display.display(msg, color=color)
 
     def v2_runner_on_skipped(self, result):
-        if self._plugin_options.get('display_skipped_hosts', C.DISPLAY_SKIPPED_HOSTS):  # fallback on constants for inherited plugins missing docs
```

Comment:
Why not keep `C.DISPLAY_SKIPPED_HOSTS`? This breaks backwards compatibility, I think, because now the behavior of callback plugins derived from `default.py` can no longer be changed with the same env variables as the behavior of the default plugin itself.

Reply:
It should still work this way via the plugin config information defined in `lib/ansible/utils/module_docs_fragments/default_callback.py`.

Reply:
Using the `DISPLAY_SKIPPED_HOSTS` env var still seem to work as expected with this PR.

Reply:
It will work with the default plugin, yes, but will it also work with derived plugins? For these, no longer `C.DISPLAY_SKIPPED_HOSTS` is used but `DEFAULT_DISPLAY_SKIPPED_HOSTS`, whose value cannot be changed. Or do I miss something?

Reply:
That's true, but that's also the case with the new configuration options introduced by my original PR. To be properly functional, a derivative callback plugin needs to include the docs snippet or copy its content.

Reply:
But that's the thing I don't understand. Why do you want to break backwards compatibility? Is there a good reason for doing it?

Reply:
I made the change for consistency, but I see your point.

Reply:
Sounds good! The newest version looks good to me. And it also still works (just re-verified) ;)